indexers/src/handlers/events/evmHost/postRequest.event.handler.ts

```diff
@@ -0,0 +1,59 @@
+import assert from "assert";
+import { RequestStatus, SupportedChain } from "../../../types";
+import { getEvmChainFromTransaction } from "../../../utils/chain.helpers";
+import { PostRequestEventLog } from "../../../types/abi-interfaces/EthereumHostAbi";
+import { HyperBridgeService } from "../../../services/hyperbridge.service";
+import { RequestService } from "../../../services/request.service";
+import { RelayerService } from "../../../services/relayer.service";
+
+/**
+ * Handles the PostRequest event from Evm Hosts
+ */
+export async function handlePostRequestEvent(
+  event: PostRequestEventLog,
+): Promise<void> {
+  assert(event.args, "No handlePostRequestEvent args");
+  logger.info("Handling PostRequest event");
+
+  const { transaction, blockNumber, transactionHash, args } = event;
+  let { data, dest, fee, from, nonce, source, timeoutTimestamp, to } = args;
+
+  const chain: SupportedChain = getEvmChainFromTransaction(transaction);
+
+  Promise.all([
+    await HyperBridgeService.handlePostRequestOrResponseEvent(chain, event),
+    await RelayerService.handlePostRequestOrResponseEvent(chain, event),
```

Comment:
The relayer does not take any from this event. This is an event initiated by a user transaction.

Reply:
Done